app/graphql/types/user_type.rb

```diff
@@ -5,5 +5,12 @@ class UserType < Types::BaseObject
 		field :last_name, String, null: false
 		field :email, String, null: false
 		field :google_token, String, null: false
+		field :activities, [String], null: true
+
+		def activities
+			object.activities.map do |activity|
+				activity.name
+			end
```

Comment:
Good incorporation of nested user's activities. Will we be doing this for categories as well?